docs/technical-design.md

```diff
@@ -142,55 +141,51 @@ The following locations are identified for the storage of data associated with A
+Core autonomous entities in the platform have an **Agent** that represents that entity in platform interactions. 
 
-Critically, all identities (SSI) created by the platform are managed by the platform on behalf of the user i.e. acting as a proxy for actions authorised by the user via their Account. The goal is to have these identities held outside of the platform - but that is in itself a journey with multiple steps on the way.
+Entities with Agents currently are: User, Organisation, Challenge, Ecoverse & Opportunity. 
 
-Worth noting that this hybrid approach also allows platform users to leverages some of the benefits of Self Sovereign Identity (SSI) such as pairwise unique identifiers etc for peer interactions, albeit with the Ecoverse as a trusted party until that trust role is decentralised. 
+Each Agent has _Credentials_ which can interact on behalf of the user / entity with other elements of the platform. 
 
-As aspects of SSI mature the intention is to allow more of the platform interaction and management to be carried out using SSI mechanisms e.g. claims, signing etc. 
+The usage of Agents is currently largely invisible to Users - but it is critical to setup the platform from the start with Agent based interaction patterns to enable moving more decentralized later.
 
-The ideal scenario for Users is that ultimately they can have their own SSI that can then control the Agents managed by the platform - or replace the platform Agents. 
+Note: the text below primarily talks about Users for Authentication and Authorization, but the approach is also applicable for other entities with Agents e.g. Challenges, Organisations etc. 
 
 ## Authentication Providers
-To authenticate users, the platform uses the concept of Authentication Providers. An Authentication Provider has the sole responsibility of authenticating users - so that upon authentication the platform knows exactly which User has logged in.
-
-<p>
-<img src="images/security-authentication-providers.png" alt="Security: Authentication Providers" width="600" />
-</p>
+The Authentication of users is handled by Authentication Providers, a pluggable mechanism whereby multiple types of Authentication can be supported by the platform. After succesful Authentication, the platform retrieves the Agent for the User and that Agent is then used to carry out actions on behalf of the User. 
 
-The currently supported Authentication Providers are:
-* Azure Active Directory (AAD) - a production, enterprise class Identity Provider from Microsoft.
-* Demo Authentication Provider - a non-production authentication provider to show the capabilities of the platform in a lightweight way.
+The platform currently supports [Ory Kratos and Ory Oathkeeper](https://ory.sh) for Authentication. The Ory platform is an open source, enterprise class Identity Provider. Note that the platform supports OAuth2 so that it is straightforward to integrate external authentications, e.g. Github, Google etc, into the platform. 
```

Comment:
In the particular case it's correct to say it uses OIDC and not OAuth.

Reply:
Fixed in 35e6d9cd0dc2687b5eb1d60b850d806a43162dab